### ISSUE TITLE
Step Values in Decimal

### ISSUE THREAD

**Issue by AnirbanKundu** [danielcrisp/angular-rangeslider#71]
Hi,

Just wanted to check if the "step" values can be given in decimal. I have "step" values for few sliders ranging from 0.1 to 0.5. I set the value to 0.1 but looks like it is not working. Any idea on how to resolve this ?

Thanks,
Anirban

**Comment by danielcrisp** [comment]
I think it should work with a decimal but I've just tried your scenario and it isn't working for me either... I suspect it might be a rounding issue

**Comment by jdmarsh** [comment]
Adding the attribute decimal-places="1" to the range slider resolves this for me

**Comment by romain10009** [comment]
yes i did the same.

**Comment by christrude** [comment]
Yeah if you set decimal places it works, but you can't have decimal places be dynamic it seems. which stinks.

**Comment by cmperez** [comment]
I'm trying to create several sliders with ng-repeat. When I try to dynamically set the decimal-places (some need to be 0, some need to be 1), none of sliders step by the correct increment.

Even when I specifically set decimal-places to 0 for each slider, the step gets overwritten, defaulting to 1.

**Comment by j-cornelio** [comment]
HI THERE

how about for smaller decimal values.  like from 0 - 1 ( 0.3, 0.5, 0.8 )

**Comment by ccdle12** [comment]
Hi,
 **decimal-places="1"** 
Will work for a range of 0 -1, so you will see 0.9 ,0.8, 0.7 etc... when moving th max slider down
